tests/Aspire.Hosting.Tests/Schema/SchemaTests.cs

```diff
@@ -266,6 +311,93 @@ public async Task ManifestWithContainerResourceAndNoEnvOrBindingsIsAccepted()
         Assert.True(manifestJson.IsValid(schema));
     }
 
+    [SkipOnHelixFact]
```

Comment:
Why are these skipped? Is it because we need the json schema file? If so, we can solve that like we do in the components tests, by copying the json file to the test output directory.

For exampe, see

https://github.com/dotnet/aspire/blob/c294706ce6276d96a0271d9f7b1bb496c72fe165/tests/Aspire.Npgsql.Tests/Aspire.Npgsql.Tests.csproj#L10